### ISSUE TITLE
Explain why tf.nn.softmax isn't used for entropy

### ISSUE THREAD

**Issue by mrahtz** [vahidk/EffectiveTensorflow#13]
Hello!

Thank you for the wonderful guide! There's one thing I'm confused about: the recipe for entropy uses a manually-defined softmax function instead of `tf.nn.softmax`. Is there a particular reason for this, or was it just to demonstrate how to implement both numerically-stable softmax and entropy?

Cheers!

**Comment by vahidk** [comment]
You can use tf.nn.softmax as well, my implementation just allows you to provide a list of dimensions to normalize across instead of a single dim.

**Comment by mrahtz** [comment]
Ahh, makes sense. Thanks!
